### PR TITLE
Fix 769: building the test binary includes all vendor package

### PR DESCRIPTION
**Pull request by cyrilleverrier**
"go test -c -covermode=atomic -coverpkg" is called with a list of packages isntead of "./..."

### CODE REVIEW

File: libbeat/scripts/Makefile [hunk 1]
```diff
@@ -35,7 +38,7 @@ TEST_ENVIRONMENT?=false
 SYSTEM_TESTS?=false
 GOX_OS?=linux darwin windows solaris freebsd netbsd openbsd
 DOCKER_COMPOSE?=docker-compose -f docker-compose.yml
-
+GOPACKAGES_COMMA_SEP=$(subst $(space),$(comma),$(strip ${GOPACKAGES}))
```
Comment (ruflin) [inline libbeat/scripts/Makefile]: Looks like quite a bit of magic. I didn't test if the line works but as the builds still work, LGTM.

**Comment by ruflin** [comment]
LGTM. Appveyor failor can be ignored as it is coming from an other PR and is already fixed.

@andrewkroh Can you have a look at this too?

File: libbeat/scripts/Makefile (hunk 1 above)
Comment (andrewkroh) [inline libbeat/scripts/Makefile]: Magic indeed. LGTM

**Comment by andrewkroh** [comment]
What's with the [`ImportError: No module named yaml`](https://ci.appveyor.com/project/elastic-beats/beats/build/837/job/1lqi27deowsmphsb#L173) error on AppVeyor? That's the first time I've seen that.

**Comment by ruflin** [comment]
It has to do with the "unification" of all builds. It was missing as it was not required before, but now is added to appveyor.yml: https://github.com/elastic/beats/pull/765/files

**Pull request merged**